napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -816,6 +819,7 @@ def _add_to_installed(distname, enabled, npe_version=1):
             )
 
         pm2 = PluginManager.instance()
+        pm2.discover()
```

Comment:
this line is to show the freshly installed plugins in the upper list. But still, installing npe2 does not cause popup with information that reinstallation is required.

Reply:
yea, freshly installed npe2 don't show for me in the top list and stay in the bottom. No pop-up about restarting—although restarting does fix it.
Non-npe2 are fine.